source/Octopus.Cli/Util/RepositoryExtensions.cs

```diff
@@ -6,16 +6,14 @@
 using Octopus.Cli.Infrastructure;
 using Octopus.Client.Exceptions;
 using Octopus.Client.Extensibility;
-using Octopus.Client.Logging;
 using Octopus.Client.Model;
 using Octopus.Client.Repositories.Async;
+using Serilog;
 
 namespace Octopus.Cli.Util
 {
     public static class RepositoryExtensions
     {
-        private static readonly ILog Logger = LogProvider.GetLogger(typeof(RepositoryExtensions));
```

Comment:
This was using OctopusClient's internal LibLog, which it shouldn't have been (LibLog is meant to stay internal to a library). Changed to use Serilog, as this is what is used by the rest of Octopus.Cli